CONTRIBUTING.rst

```diff
@@ -54,7 +52,7 @@ For other questions, discussions, and chats, we have:
   This is a busy place with a lot of CI and commit notifications that makes
   actual chat sometimes difficult!
 
-- a mailing list at `sourceforge <https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss>`_
+- a mailing list at `SourceForge <https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss>`_
```

Comment:
I couldn't find anything at https://lists.sourceforge.net/lists/listinfo/aboutcode-discuss, this is old and maybe we want to remove this?

Reply:
Is there an alternative, or should this be removed entirely?
 I've removed it for the time being, but if any new mailing lists appear, I'll update them.